apps/frontend/src/routes/blog.$blogId.lazy.tsx

```
@@ -0,0 +1,81 @@
+                                }
+                            )}
+                        </time>
+                    </section>
+
+                    <figure
+                        className={`w-lvw max-w-prose place-self-center ${!post.image && 'hidden'}`}
+                    >
+                        <img
+                            src={post.image || undefined}
+                            className="w-full object-contain sm:rounded-2xl"
+                        />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)